shared/reducers/crypto.tsx

```diff
@@ -27,6 +27,8 @@ export default Container.makeReducer<Actions, Types.State>(initialState, {
 
     if (operationGuard(operation, action)) return
 
+    draftState[operation].bytesComplete = 0
+    draftState[operation].bytesTotal = 0
```

Comment:
@cjb What happens if the service continues to push progress after the input is cleared?

E.g. Encrypting an `X`Gb file and then clear the input (effectively canceling) before the file operation is complete?

Reply:
Dunno! We could remove the zeroing from clearRecipients if it doesn't work well.  We don't have good ordering guarantees (e.g. we get Progress after Done sometimes) so perhaps just displaying the latest Progress output all the time is going to look best..

Reply:
If the ordering is a problem, I can make the notifications more synchronous.

Reply:
Hmm, perhaps cancelling a file operation is something I didn’t fully consider. Especially for large files.

Regardless of the progress notifications being synchronous, it seems that halting an saltpack operation on a file is impossible currently.